buildworker.rst

```diff
@@ -111,7 +111,64 @@ machine reboots:
 
 For Linux:
 
-    * Add the following line to ``/etc/crontab``::
+    * For systemd based distributions, you can create a systemd unit file in order
+      to manage the service. Create the unit file named ``buildbot-worker.service``
+      under ``/etc/systemd/system`` and change the paths according to where the
+      buildbot-worker binary resides. You can verify its location by running
+      ``which buildbot-worker``. If you installed the buildbot-worker through
+      your package manager it would be::
+
+         [Unit]
+         Description=Buildbot Worker
+         Wants=network.target
+         After=network.target
```

Comment:
The builder should wait for network online and start after network is online and all local file systems are available:

```
Wants=network-online.target
After=network-online.target local-fs.target
```

Reply:
Will amend that